src/main/java/com/ebay/myriad/scheduler/MyriadDriver.java

```diff
@@ -46,7 +46,9 @@ public MyriadDriver(final MyriadScheduler scheduler,
         Builder frameworkInfoBuilder = FrameworkInfo.newBuilder().setUser("")
                 .setName(cfg.getFrameworkName())
                 .setCheckpoint(cfg.getCheckpoint())
-                .setFailoverTimeout(cfg.getFrameworkFailoverTimeout());
+                .setFailoverTimeout(cfg.getFrameworkFailoverTimeout())
+                .setWebuiUrl(cfg.getWebuiURL());
```

Comment:
Since you're allowing a hostname override, you should also setHostname (assuming it's different from your default "localhost"). If left empty/unspecified, the MesosSchedulerDriver will default to `net::hostname()`.